corehq/apps/groups/templates/groups/all_groups.html

```diff
@@ -35,6 +35,16 @@
 {% endblock %}
 
 {% block main_column %}
+{% if needs_to_downgrade_locations %}
+    <div>
+        <p>
+            {% trans "This project no longer has access to the Organization feature." %}
+            <a href="{% url "downgrade_locations" domain %}">
+                {% trans "Click here to fix this." %}
```

Comment:
Can you put the controls for downgrading locations on this page instead of linking to a different page? As is, this page is basically nothing but a link, not great for performance.